Dockerfile

```diff
@@ -0,0 +1,28 @@
+# build stage
+FROM node:13-slim AS build-env
+COPY . .
+RUN rm -rf node_modules
```

Comment:
I think instead of doing this, just add a `.dockerignore` file with node_modules in it.